CHANGELOG.md

```diff
@@ -1,3 +1,91 @@
+Version 1.13.0
+==============
+
+This is a minor version release packed full of features and a general
```

Comment:
```suggestion
This release is packed full of features and includes a general
```

(I think it's confusing to call this a minor release, even if that's what the version number says; really this is a major release.)

Reply:
Applied suggestion in 7869f5632